Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/Definitions/Enums.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel;
+#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    [DefaultValue("UTF-8")]
+    public enum Encode
```

Comment:
Please add XML-documentation about this enum, so it will automatically generated in the Tasks Portal.